File: 2025_04/leetcode2025-04-30.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int res = 0;
        for (int i : nums){
            if (to_string(i).size()%2 == 0){
                res++;
            }
        }
        return res;
    }
};

int main(){
    Solution s;
    vector<int> nums = {12,345,2,6,7896};
    cout << s.findNumbers(nums) << endl;
}

File: 2025_04/leetcode2025-04-30.py
from typing import List

class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        return sum(1 for i in nums if len(str(i))%2==0)

if __name__ == "__main__":
    s = Solution()
    nums = [12,345,2,6,7896]
    print(s.findNumbers(nums))
